ci/azure/linux.yml

```diff
@@ -73,15 +62,17 @@ jobs:
         docker-compose run \
           -e PYTHONHASHSEED=$PYTHONHASHSEED \
           -e AZURECI=$AZURECI \
+          -e PYTEST_BACKENDS="$PYTEST_BACKENDS" \
           ibis \
-          pytest ibis -m "${PYTEST_MARK_EXPRESSION}" \
-                      -k"not compile and not connect" \
-                      -ra \
-                      --doctest-modules \
-                      --doctest-ignore-import-errors \
-                      --junitxml=/tmp/test-reports/pytest/junit.xml \
-                      --cov=ibis \
-                      --cov-report=xml:/tmp/test-reports/pytest-cov/coverage.xml
+          pytest ibis/tests ibis/expr/tests ibis/sql/tests ibis/impala/tests ibis/clickhouse/tests \
+            -m "not udf" \
```

Comment:
shoudn't this skip int he tests themselves? (the udf)

Reply:
Not sure I understand. Do you mean that the udf tests could be skipped depending on the backend with the decorators as the rest of the tests? Probably, but it's been like this until now, and I guess some extra refactoring is needed. I can open an issue if this is what you mean. Not so sure if makes sense for hdfs.

Reply:
yes that's exactly what i mean, e.g. the CI should simply run the backend w/o *any* logic for which marks it should run / not run (these are in the code). but yeah let's open an issue and not block this PR.